examples/python/route_guide/run_codegen.py

```diff
@@ -28,5 +27,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Runs the protoc with gRPC plugin to generate protocol messages and gRPC stubs.
-python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto
```

Comment:
The motivation for this change is that the tutorial lends itself better to being used with `virtualenv`. Without this change, the shell script would probably have to be updated to accept as an argument the particular `python` to execute.

@hsaliak and I talked about this; I think he had some other reasons as well.

[This pull request](https://github.com/grpc/grpc.github.io/pull/223) by @LisaFC is one of the documentation updates to which you allude, right?